src/Query/User/PackageQuery/DbalPackageQuery.php

```diff
@@ -144,7 +143,7 @@ public function versionCount(string $packageId): int
     /**
      * @return Version[]
      */
-    public function getVersions(string $packageId, int $limit = 20, int $offset = 0): array
+    public function getVersions(string $packageId, \Buddy\Repman\Query\Filter $filter): array
```

Comment:
I wonder if we can somehow automate the detection of this notation?

Reply:
Possibly 'fully_qualified_strict_types', but it looks like that only works when the use statement is there in the first place